web/src/components/ContrastThemeToggle.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { ContrastIcon } from '../assets'
+import { useContrastTheme } from '../hooks/useContrastTheme'
+import useWindowDimensions from '../hooks/useWindowDimensions'
+import ToolbarItem from './ToolbarItem'
+import Button from './base/Button'
+import Icon from './base/Icon'
+
+const ContrastButton = styled(Button)`
+  display: flex;
+  width: 100%;
+  padding: 24px 0;
+  align-items: center;
+  cursor: pointer;
+
+  & > span {
+    padding: 0 28px;
+    color: ${props => props.theme.colors.textColor};
+  }
+`
+
+const StyledIcon = styled(Icon)<{ $isContrastTheme: 'light' | 'contrast' }>`
+  width: 24px;
+  height: 24px;
+  transform: ${({ $isContrastTheme }) => ($isContrastTheme === 'contrast' ? 'reverse' : 'none')};
+  transition: transform 0.2s ease;
+`
+
+const ContrastThemeToggle = (): ReactElement => {
+  const { t } = useTranslation('layout')
+  const { viewportSmall } = useWindowDimensions()
+  const { toggleTheme, themeType } = useContrastTheme()
+
+  if (viewportSmall) {
+    return (
+      <ContrastButton dir='auto' label={t('contrastTheme')} onClick={toggleTheme}>
```

Comment:
🙃 `dir='auto'` is the default, so no need to pass it explicitly:
```suggestion
      <ContrastButton label={t('contrastTheme')} onClick={toggleTheme}>
```